course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -19,8 +19,8 @@
 from course_discovery.apps.course_metadata.data_loaders import AbstractDataLoader
 from course_discovery.apps.course_metadata.data_loaders.course_type import calculate_course_type
 from course_discovery.apps.course_metadata.models import (
-    Course, CourseEntitlement, CourseRun, CourseRunType, CourseType, Organization, Program, ProgramType, Seat, SeatType,
-    Subject, Video, Person, PersonSocialNetwork
+    Course, CourseEntitlement, CourseRun, CourseRunType, CourseType, Organization, Person, PersonSocialNetwork, Program,
+    ProgramType, Seat, SeatType, Subject, Video
```

Comment:
Put every item in new line.

Reply:
Quality checks raise an error when every item added in a new line 

![image](https://user-images.githubusercontent.com/7139602/110894929-99f09000-831a-11eb-9c29-c3e8e1edd280.png)